src/tailwind.js

```diff
@@ -13,6 +13,14 @@ const localConfig = {
     ],
   },
   theme: {
+    backgroundImage: {
+      "single-wave": 'url("/static/images/home/mainBenefits/single-wave.png")',
+      map: 'url("/static/images/home/welcome/map.png")',
+      "wave-lines": 'url("/static/images/common/wave-lines.png")',
+      "news-generic": 'url("/static/images/news/news-generic.png")',
+      "waves-lines-light": 'url("/static/images/common/wave-lines-light.png")',
+      "waves-lines-light-2": 'url("/static/images/common/wave-lines-light-2.png")',
+    },
```

Comment:
imo i would retain inline way within the component itself, will only consolidate here provided it is commonly reused more than once across the app. wdyt?

Reply:
yeap, followed as you mentioned, but like a bit leceh, cos like need check if it the background image exist before inlining.

waves-lines-light is mentioned thrice and single-wave once, but it's needed so that can use the lg: tag